setup/__tests__/find-haskell.test.ts

```diff
@@ -55,43 +48,34 @@ describe('haskell/actions/setup', () => {
       forAllTools(t => expect(def(os)[t].supported).toBe(supported_versions[t]))
```

Comment:
One thing I missed the first time through is now that we're not doing weird things to the ENV, lines 33-40 don't need to exist anymore.

```javascript
  const OLD_ENV = process.env;

  beforeEach(() => {
    jest.resetModules();
    process.env = {...OLD_ENV};
    delete process.env.NODE_ENV;
  });

  afterEach(() => (process.env = OLD_ENV));
```

I won't be sad to see this code go in the slightest :)